homeassistant/components/switcher_kis/__init__.py

```diff
@@ -77,3 +79,12 @@ async def stop_bridge(event: Event | None = None) -> None:
 async def async_unload_entry(hass: HomeAssistant, entry: SwitcherConfigEntry) -> bool:
     """Unload a config entry."""
     return await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+
+
+async def async_remove_config_entry_device(
+    hass: HomeAssistant, config_entry: SwitcherConfigEntry, device_entry: dr.DeviceEntry
+) -> bool:
+    """Remove a config entry from a device."""
+    return not device_entry.identifiers.intersection(
+        (DOMAIN, device_id) for device_id in config_entry.runtime_data
```

Comment:
Why don't we remove these devices automatically during config entry setup?

Reply:
We don't know during setup the status of the device. Devices are added during runtime when we get a broadcast from them

Reply:
Ok.